### PR TITLE
Add justification template

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

**Comment by bissenbay** [comment]
@fridex 
```
No .thoth.yaml was provided.
Please add .thoth.yaml to your PR.
```
:smile:

File: _justifications/example.md.template [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Justification title
```
Comment (goern) [inline _justifications/example.md.template:2]: doesnt a justification have some kind of severity? like WARN, ERROR ?!

File: _justifications/example.md.template [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis
+nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
+fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+culpa qui officia deserunt mollit anim id est laborum.
+
+## Pipeline units
+
```
Comment (goern) [inline _justifications/example.md.template:15]: can we link to the implementation?

File: _justifications/example.md.template [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+ * TensorFlowAVX2Steps
+
+## Recommendation types
```
Comment (goern) [inline _justifications/example.md.template:18]: do we have a piece of doc describing what a recommendation type is?

**Review by goern — CHANGES_REQUESTED** [review]

File: _justifications/example.md.template (hunk 1 above)
Comment (fridex) [reply _justifications/example.md.template:2]: +1, can be added

File: _justifications/example.md.template (hunk 2 above)
Comment (fridex) [reply _justifications/example.md.template:15]: yes, it was meant like that, I'll probably adjust the example

File: _justifications/example.md.template (hunk 3 above)
Comment (fridex) [reply _justifications/example.md.template:18]: We can create it.

**Comment by fridex** [comment]
I would like to merge this to restructure links. All comments were resolved, we will create a document for recommendation types. Please raise any comments/craete issues.

**Pull request merged**